roadmap.md

```diff
@@ -16,7 +16,7 @@ milestone. The convention for the priorities are:
 *   P0 feature will block the milestone; we will delay the milestone date
```

Comment:
Do you still want to keep all this talk about milestones?

Reply:
I was wondering, I kept it since we still have priorities for 1.0 and for the LTS.

Reply:
Ok, it makes sense, but then update the very first sentence of this doc that reads: "This document describes the Bazel team's plans for ... version 1.0". Drop 1.0 from there.

Reply:
Done.